packages/cursorless-engine/src/actions/FollowLink.ts

```diff
@@ -21,15 +26,7 @@ export default class FollowLink implements SimpleAction {
 
     const openedLink = await ide()
```

Comment:
This return value is not used anymore. Should probably be removed from the openLink type.